tests/test_rabbit_context.py

```diff
@@ -2,10 +2,10 @@
 from unittest.mock import patch
 
 from app import create_app
-from app.rabbit_context import RabbitContext, RabbitConnectionClosedError
+from rabbit_context import RabbitContext, RabbitConnectionClosedError
```

Comment:
I don't think these unit tests need to be here anymore, the code they're testing doesn't live in this repo now.